packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -115,21 +117,21 @@ export interface DropdownProps
   inline?: boolean
 
   /** Array of props for generating list options (Dropdown.Item[]) and selected item labels(Dropdown.SelectedItem[]), if it's a multiple selection. */
-  items?: ShorthandCollection
+  items?: ShorthandCollection<DropdownItemProps>
```

Comment:
This is also use in the `DropdownSelectedItem.create` factory. How should we handle this one?

Reply:
I don't have a good idea, left a comment as we discussed offline